src/common/collection.c

```diff
@@ -40,7 +40,7 @@
 
 
 #define SELECT_QUERY "SELECT DISTINCT * FROM %s"
-#define ORDER_BY_QUERY "ORDER BY %s"
+#define ORDER_BY_QUERY "ORDER BY" /*changed format for second order sorting!*/
```

Comment:
Well since ORDER_BY_QUERY is now a simple string...

Reply:
done.